app/controllers/wpcc/your_details_controller.rb

```diff
@@ -8,11 +10,13 @@ def new
 
     def create
       @your_details_form = Wpcc::YourDetailsForm.new(your_details_form_params)
+      @your_details_form = present(@your_details_form)
```

Comment:
@willhall88 @margOnline putting it all on online will break the 80char Rubocop limit. I'll return it to being a single assignment, but with a different way to break 'em into multiple lines.